lib/src/components/NxToast/NxToast.scss

```diff
@@ -5,28 +5,21 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-toast-wrapper {
-  // nx-toast-wrapper has position absolute to ensure toasts stay in the top right corner of the page.
-  // Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
-  // nx-page-main. The nx-toast-container has position fixed applied so it remains fixed while scrolling.
-  position: absolute;
-  top: 0;
-  right: 0;
-  display: flex;
-  justify-content: flex-end;
-  margin-right: var(--nx-spacing-6x);
-}
-
 .nx-toast-container {
-  position: fixed;
+  position: sticky;
+  left: calc(100% - 500px);
+  height: max-content;
   width: 500px;
+  z-index: 1;
```

Comment:
As I mentioned in another comment, there will need to be coordination between toasts and `NxDrawer`.  You should work with @rjp99 to make sure you pick a `z-index` that will result in the toasts appearing in front of drawers.

Reply:
> As I mentioned in another comment, there will need to be coordination between toasts and `NxDrawer`. You should work with @rjp99 to make sure you pick a `z-index` that will result in the toasts appearing in front of drawers.

I reached out and they have NxDrawer currently set to 1040, so I set toast to 1050 here: [1113705](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/1113705626690b1b17271c6beb56615f80433488)